lib/inspector/diagnostics_node.dart

```diff
@@ -345,7 +356,10 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
     if (value == null) {
       return defaultValue;
     }
-    final level = diagnosticLevelNames[value];
+    final level = nameToDiagnosticLevel[value];
+    if (level == null) {
+      print ('Unabled to find level for $value');
```

Comment:
remove this debugging print and instead add a string message to the assert.

Reply:
👍